### ISSUE TITLE
`// ssssswiftlint:disable rule_name` (or any other prefix to `swiftlint:`) is a valid command

### ISSUE THREAD

**Issue by mildm8nnered** [realm/SwiftLint#4789]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

So normally commands look like `// swiftlint:disable some_rule`, or `// swiftlint:enable some_rule`, and they can have modifiers etc., but when parsing commands, we just look for `swiftlint:`, which means that as long as we're in a comment, any arbitrary string can be prepended on, and the command will still work - e.g.

`// ghjhkjklhjkhjkljkljhkhswiftlint:disable some_rule`

which is not terrible, but feels a bit weird.

Not sure what should happen here really - maybe a warning, which would preserve existing behaviour, unless it's a comment character or whitespace preceding `swiftlint:`

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
% cat Foo.swift 
// hikjkljkhjkljkljklswiftlint:disable comment_spacing
//jkljkl
% swiftlint Foo.swift
Linting Swift files at paths Foo.swift
Linting 'Foo.swift' (1/1)
Done linting! Found 0 violations, 0 serious in 1 file.
```

Without the disable, this generates

```
% swiftlint Foo.swift
Linting Swift files at paths Foo.swift
Linting 'Foo.swift' (1/1)
/tmp/Foo.swift:1:3: warning: Comment Spacing Violation: Prefer at least one space after slashes for comments. (comment_spacing)
Done linting! Found 1 violation, 0 serious in 1 file.
```

### Environment

* SwiftLint version? 0.50.3
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Homebrew
* Paste your configuration file:

none

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  If so, paste their relative paths and respective contents.

no

* Which Xcode version are you using (check `xcodebuild -version`)?

Xcode 14.2
Build version 14C18

* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// hikjkljkhjkljkljklswiftlint:disable comment_spacing
//jkljkl
```

Does not trigger a violation for comment_spacing

**Comment by mildm8nnered** [comment]
Resolved by #5204
